### PR TITLE
Add workflow to approve dependabot PRs

### PR DESCRIPTION
**Pull request by defunctzombie**
### Changelog
None

### Docs
None

### Description
Auto-approve and merge dependabot PRs.

### CODE REVIEW

**Comment by defunctzombie** [comment]
@jtbandes @achim-k the linear issue asks for auto-merge as well. I can add that if we think it would be safe and appropriate?

File: .github/workflows/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+jobs:
+  approve:
+    name: Approve Dependabot PRs
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+
+    # IMPORTANT: setting YARN_ENABLE_SCRIPTS=false is critical to ensure that untrusted
+    # PRs can't add an npm package and then use that to execute untrusted code in
+    # a trusted context. See links at the top of this workflow for further details.
```
Comment (achim-k) [inline .github/workflows/dependabot.yml:16]: Outdated commented? There are no links at the top of this workflow

**Review by achim-k — APPROVED** [review]

**Comment by achim-k** [comment]
Auto-merge would be useful. I believe that we have enough tests in place to make it safe enough

File: .github/workflows/dependabot.yml (hunk 1 above)
Comment (defunctzombie) [reply .github/workflows/dependabot.yml:16]: Yea - the links part is outdated but it seemed nice to keep this env var as a precaution? Though this PR doesn't run yarn anymore now that I think about it

**Review by jtbandes — APPROVED** [review]
Seems alright. Will point out that we don't seem to have any tests for FoxgloveClient right now.

**Comment by defunctzombie** [comment]
> Will point out that we don't seem to have any tests for FoxgloveClient right now.

Perfect 👏

**Pull request merged**